en/topics.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Topics
+permalink: /en/topics/
+layout: page
+---
+{% include linkers/topic-pages.md %}
```

Comment:
This centered element looks a little odd when the title and content are both left-aligned. My recommendation would be to center-align the title and have a dynamic number of columns for the list below.